docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -83,51 +84,38 @@ text_file      | [text_file][]      | Configures the text_file collector.      |
-
-- `ADAccessProcesses`
-- `TransportQueues`
-- `HttpProxy`
-- `ActiveSync`
-- `AvailabilityService`
-- `OutlookWebAccess`
-- `Autodiscover`
-- `WorkloadManagement`
-- `RpcClientAccess`
-
-For example, `enabled_list` may be set to `"AvailabilityService,OutlookWebAccess"`.
-
+Name           | Type           | Description                  | Default       | Required
+---------------|----------------|------------------------------|---------------|---------
+`enabled_list` | `list(string)` | A list of collectors to use. | `["ADAccessProcesses", "TransportQueues", "HttpProxy", "ActiveSync", "AvailabilityService", "OutlookWebAccess", "Autodiscover", "WorkloadManagement", "RpcClientAccess", "MapiHttpEmsmdb"]` | no
 
 ### iis block
 
-Name           | Type     | Description                                      | Default | Required
----------------|----------|--------------------------------------------------|---------|---------
-`app_exclude`  | `string` | Regular expression of applications to ignore.    | `""`    | no
-`app_include`  | `string` | Regular expression of applications to report on. | `".*"`  | no
-`site_exclude` | `string` | Regular expression of sites to ignore.           | `""`    | no
-`site_include` | `string` | Regular expression of sites to report on.        | `".*"`  | no
+Name           | Type     | Description                                      | Default   | Required
+---------------|----------|--------------------------------------------------|-----------|---------
+`app_exclude`  | `string` | Regular expression of applications to ignore.    | `"^$"`    | no
+`app_include`  | `string` | Regular expression of applications to report on. | `"^.+$"`  | no
+`site_exclude` | `string` | Regular expression of sites to ignore.           | `"^$"`    | no
+`site_include` | `string` | Regular expression of sites to report on.        | `"^.+$"`  | no
 
+User-supplied `app_exclude`, `app_include`, `site_exclude` and `site_include` strings will be [wrapped][wrap-regex] in a regex.
 
 ### logical_disk block
 
-Name      | Type     | Description                               | Default | Required
-----------|----------|-------------------------------------------|---------|---------
-`exclude` | `string` | Regular expression of volumes to exclude. | `""`    | no
-`include` | `string` | Regular expression of volumes to include. | `".+"`  | no
+Name      | Type     | Description                               | Default   | Required
+----------|----------|-------------------------------------------|-----------|---------
+`exclude` | `string` | Regular expression of volumes to exclude. | `"^$"`    | no
+`include` | `string` | Regular expression of volumes to include. | `"^.+$"`  | no
 
 Volume names must match the regular expression specified by `include` and must _not_ match the regular expression specified by `exclude` to be included.
 
+User-supplied `exclude` and `include` strings will be [wrapped][wrap-regex] in a regex.
```

Comment:
@mattdurham @clayton-cornell @jkroepke Are you happy with this explanation? And would you be ok with me raising a "v2 breaking change" issue for removing this regex wrapping?

Reply:
If you ask me, I would link against every `Regular expression` on each Description to make it clearly visible.

Reply:
That's a good idea, since we don't expect the wrapping to get in the way of users. It should be invisible, but we're documenting just as an FYI to users in case they experience something weird. @clayton-cornell, WDYT?

Reply:
Linked the way it is now it's fairly clear to me on reading it all through and following the link to the regex explanation...

Reply:
I like how the current explanation clearly states "this is wrapped in this", and I wish I could say it clearly in the table itself in a concise way. I can't think of a concise way to do it though, so let's just merge it as it is since it's more clear.